ui/js/controllers/jobs.js

```diff
@@ -90,13 +90,13 @@ treeherderApp.controller('ResultSetCtrl', [
     'thUrl', 'thServiceDomain', 'thResultStatusInfo', 'thDateFormat',
     'ThResultSetStore', 'thEvents', 'thJobFilters', 'thNotify',
     'thBuildApi', 'thPinboard', 'ThResultSetModel', 'dateFilter',
-    'ThModelErrors', 'ThJobModel', 'ThTaskclusterErrors', '$uibModal',
+    'ThModelErrors', 'ThJobModel', 'ThTaskclusterErrors', '$uibModal', 'pinboardError',
```

Comment:
I am wondering if this should be more specific, like **thPinboardCountError** or **thPinboardOverflowError** since there are/will-be other kinds of pinboard errors which are not count related. Again I defer to @camd and note I haven't tested this branch yet. :)

I also don't know myself about the case-ness of `Th` vs. `th` in the items above. Maybe there is some rationale when each is used and Cameron knows that too.

Reply:
A good point, @tojon : A more exact name would be better.

The thing about the ``Th`` prefix and ``th`` prefix goes WAY back and I'm not sure it really matters, tbh.  I suppose it's nice.  The capital vs lower case was: Capital for models and model-type things.  Lower case for other things.

It was a way we thought would help distinguish what the piece belongs to.  If it doesn't have a ``th`` prefix, then it's external.  Or some have a ``ph`` for Perf Herder.  We may do away with this at somepoint, but for now, let's go with ``th`` prefix.  Just for consistency.  We may do away with it at some point.  :)

Reply:
That's excellent, thanks for the history on that @camd. !